command/policy_write.go

```diff
@@ -91,7 +92,9 @@ func (c *PolicyWriteCommand) Run(args []string) int {
 	}
 
 	// Policies are normalized to lowercase
-	name := strings.TrimSpace(strings.ToLower(args[0]))
+	policyName := args[0]
+	warnOnUpperCase(c.UI, policyName)
+	name := strings.TrimSpace(strings.ToLower(policyName))
```

Comment:
Maybe name this `normalizedName` or `formattedName`? to differentiate it from simple `policyName`

Reply:
I didn't even notice that variable name but you're so right, thank you!